SU2_CFD/src/fluctuating_stress.cpp

```diff
@@ -111,41 +111,90 @@ void CM43Model::CalculateEddyViscosity(const CGeometry* geometry,
   assert(eddy_viscosity != NULL);
   assert(C_M > 0);
 
+  const su2double damping = ComputeDamping(geometry, config, iPoint);
+
   /*--- C_M0 is an overall coefficient used to calibrate the model to match
    * isotropic resolution ---*/
   const su2double C_M0 = 0.13;
 
   for (unsigned short iDim = 0; iDim < nDim; iDim++) {
     for (unsigned short jDim = 0; jDim < nDim; jDim++) {
-      eddy_viscosity[iDim][jDim] =
-          density * C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
+      eddy_viscosity[iDim][jDim] = density * C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
+      //C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
+	//C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
+	//density * C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
+	//density * damping*C_M0*C_M * pow(dissipation, 1.0/3) * M43[iDim][jDim];
```

Comment:
@trevilo We calculate damping here, but we don't use it? What's the status on the damping?

Reply:
Yeah, the damping or wall-blending of the SGET term is something I messed around with, but without ever coming to a definitive conclusion for how it should be handled.  You can see evidence of this in the code just below, where as similar blending is applied as what is in CDP (although it blends to 0 rather than the RANS eddy viscosity b/c I had stability problems when blending to the RANS eddy viscosity).

In any case, Sigfried did not need this blending for the channel in CDP.  He added it later for very high aspect ratio cells in the WMH.  Empirically, I observed that we did need something (for stability) in SU2, and the stability of the code was somewhat sensitive to how exactly it was done.  However, those observations are mostly from when the SGET term was treated fully explicitly.  So, it should be less sensitive now I would hope.

Reply:
Alright.  I'm moving this to a config option, with three choices:

1. No high-AR damping
2. Blend to zero for high-AR
3. Blend to RANS for high-AR